tests/test_pyxis.py

```diff
@@ -0,0 +1,426 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020  Red Hat, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import requests
+
+from http import HTTPStatus
+from copy import deepcopy
+from unittest.mock import call, patch, create_autospec, Mock
+
+from freshmaker import conf
+from freshmaker.pyxis import Pyxis
+
+from tests import helpers
+
+
+class TestQueryPyxis(helpers.FreshmakerTestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.koji_read_config_patcher = patch(
```

Comment:
Are these two lines needed since I don't think you use the `koji` module at all in `pyxis.py`?